spec/bolt/config_spec.rb

```diff
@@ -185,6 +185,18 @@
       }.to raise_error(Bolt::ValidationError)
     end
 
+   it "does not accept a non-array for run-as-command" do
+      config = {
+        transports: {
+          ssh: { 'run-as-command' => 'sudo -n' }
+        }
+      }
+      expect {
+        Bolt::Config.new(config).validate
```

Comment:
We should probably have a positive test as well. None of the tests caught the broken validation check.